torchgeo/datasets/reforestree.py

```diff
@@ -0,0 +1,301 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ReforesTree dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional, Tuple
+
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import check_integrity, download_and_extract_archive, extract_archive
+
+
+class ReforesTree(VisionDataset):
+    """ReforesTree dataset.
+
+    The `ReforesTree <https://github.com/gyrrei/ReforesTree>`_
+    dataset contains drone imagery that can be used for tree crown detection,
+    tree species classification and Aboveground Biomass (AGB) estimation.
+
+    Dataset features:
+
+    * 105 high resolution RGB drone images at 2cm/pixel of size 4000x4000px
+    * more than 4600 tree crown box annotations
+    * tree crown matched with field measurements of diameter at breast height (DBH),
+      and computed AGB and carbon values
+
+    Dataset format:
+
+    * images are three-channel pngs
+    * annotations are csv file
+
+    Dataset Classes:
+
+    0. other
+    1. banana
+    2. cacao
+    3. citrus
+    4. fruit
+    5. timber
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * Reiersen, Gyri, et al. "ReforesTree: A Dataset for Estimating Tropical
+      Forest Carbon Stock with Deep Learning and Aerial Imagery."
+      arXiv preprint arXiv:2201.11192 (2022).
+
+    .. versionadded:: 0.3
+    """
+
+    classes = ["other", "banana", "cacao", "citrus", "fruit", "timber"]
+    url = (
+        "https://uc62e9acc213a8f56919a7a7add8.dl.dropboxusercontent.com/"
+        "zip_download_get/BKn4_obEfo0iR6UUpOqBMp9K0ohkldQtJTUN802va4SgJvd"
+        "OsN36ap5fcbbxFLbhPMHoI1oJBrkjwJWyYkcBE-SxCeFPTZ0a-gOBozXL7CfaKg?"
+        "_download_id=600502027618175772326304665489490007078769290252973"
+        "08895167120694292&_notify_domain=www.dropbox.com&dl=1"
+    )
```

Comment:
You can probably just include this on a single line, flake8 should ignore it since it's a URL

Reply:
flake8 fails on this one-liner version